### ISSUE TITLE
Not an issue, just wanted to say thanks!

### ISSUE THREAD

**Issue by Deus-nsf** [qwerty-fr/qwerty-fr#55]
This might be the best European keyboard layout available, thanks so much!

Actually, since it's the issues thread, if I really wanted to nitpick I could say the naming doesn't really reflect what it is, it goes well beyond just french characters, maybe the name "qwerty-eu" would be better?

**Comment by devnoname120** [comment]
Thanks for your warm comment!
With regard to the naming I agree that we should ultimately move away from `qwerty-fr`. I will organize a “give-your-suggestions” thread at some point.

**Comment by adn34** [comment]
I just want to echo a big thank you, I was struggling with the dead keys on international qwerty and this layout is so much more logical and convenient with the accents, only took like 15mns for it feel natural.

**Comment by Deus-nsf** [comment]
Yeah right! Still has issues between è and é because è is more common so I tend to press e to form è, but I havent been using accents a lot so far, maybe I just need to get used to it.
I'm definitely keeping the layout though!

**Comment by daks** [comment]
I'm joining people to say "thank you!" too :)
Using it on a Preonic ortholinear keyboard where I had no option except to drop the azerty layout (because of keycaps availability), qwerty-fr is so much better than other alternatives!

**Comment by Deus-nsf** [comment]
Yeah right! 
Still having trouble with "é" being on W but I have to force myself to get used to it, I can't judge so quickly.

**Comment by Deus-nsf** [comment]
Okay, so, I've been using this layout pretty much everyday, and I really wanted to have a long term feel about it.
I'm still really happy with it, but I definitely think in the end that é and è should be swapped, so é on E and è on W.
Might be worth creating a poll about it?

**Comment by adn34** [comment]
Per the documentation modifying the layout shouldn't be too hard given all the groundwork is done. On Windows at least the layout is modifiable by using the link they provide; https://www.microsoft.com/en-us/download/details.aspx?id=102134 and the "Modify an existing keyboard layout (.KLC) file and build a new layout from it" section, the KLC for qwerty-fr is right here in the github.

I don't mind giving it a try when I have some time so probably next weekend. I don't think breaking the logic of all other accents with the left -> acute and right -> grave would be worth it just for é even if I agree it feels more natural to hit the e key for it, so a personal modification or doing a fork is probably the best way to get that.

**Comment by Deus-nsf** [comment]
Oh yeah no, that would definitely be just for é (I just missed it again lol), more like an "exception" approach.
I know exceptions suck, most of the time, but for something as recurrent as é it is deserved in my opinion.

**Comment by devnoname120** [comment]
@Deus-nsf @adn34 It's not the right place to discuss the design of the layout. Please move to the discussions, thanks!
